Turkiz/Turkiz/Models/User.cs

```diff
@@ -0,0 +1,14 @@
+namespace Turkiz.Models
+{
+    public class User
+    {
+        public int UserId { get; set; }
+        public string UserName { get; set; }
+        public string Password { get; set; }
+        public string Email { get; set; }
+        public string? Phone { get; set; }
```

Comment:
This is a valid syntax?
I think you need to remove the question mark and add the [Optional] annotation above it.